superduperdb/ext/openai/model.py

```diff
@@ -316,6 +324,9 @@ class OpenAIImageEdit(OpenAI):
     takes_context: bool = True
     prompt: str = ''
 
+    def __post_init__(self):
+        self.encoder = pil_image
```

Comment:
This will break since the output of those models is currently `bytes`. So you would need to handle that or change the default output type. I would suggest the 2nd for simplicity.